RELEASE_NOTES.md

```diff
@@ -15,3 +15,16 @@ SSL connections.
 New Features
 ============
 
+* `drizzle_escape_str`
+
+    This new function matches the signature of `drizzle_escape_string`, except it takes an extra parameter
+    to determine whether to escape pattern-matching characters ('%' and '_') or not.
+    Calling `drizzle_escape_string` is equivalent to calling `drizzle_escape_str` with `0` as last parameter.
```

Comment:
You have weirdly managed to put the text in italics here but it doesn't seem to affect the final result.